src/System.Buffers/ref/Configurations.props

```diff
@@ -4,10 +4,11 @@
     <PackageConfigurations>
       netstandard;
       netstandard1.1;
-      uap10.0.16299;
+	    net45;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations);
+      uap10.0.16299;
```

Comment:
You will need to add a versionless configuration for netfx vertical in order for this to work, or else vertical netfx build won't have the assets restored correctly.